david00medina/kanpla-billbridge
# KAN-5 Installed ORM w/ SQLite and scheduler packages

### 🛠 KAN-5 Installed ORM with SQLite and Scheduler Packages

**Description:**
This PR adds the necessary packages for database integration using SQLite with TypeORM and scheduling capabilities via `@nestjs/schedule`. These additions enable database operations and scheduled task execution for the application.

**Changes Introduced:**
- Installed `typeorm`, `sqlite3`, and `@nestjs/typeorm` for database integration.
- Installed `@nestjs/schedule` for task scheduling.
- Updated `package.json` and `package-lock.json` to include the new dependencies.

**Testing Instructions:**
1. Pull the branch and run `npm install`.
2. Verify that the project starts successfully with the installed dependencies.
3. Confirm that TypeORM and scheduling modules are available for use.

**Purpose:**
To enable database persistence with SQLite and provide a lightweight scheduling mechanism for periodic tasks within the application.

---

Let me know if further configuration or adjustments are required.
